sklearn/cross_validation.py

```diff
@@ -1858,7 +1858,9 @@ def train_test_split(*arrays, **options):
         matrices or pandas dataframes.
 
         .. versionadded:: 0.16
-            preserves input type instead of always casting to numpy array.
```

Comment:
As in the previous version of this note, it's not clear that it's discussing the return value. Just delete this note.